### PR TITLE
Use HTTPS instead of protocol-relative CDN URLs

### LINKED ISSUE #86: Use https for download links
Right now we use http for downloading even if the user came in via https. Now that the CDN supports https we should use it, at the very least when the user comes in via a secure connection, by removing the protocol from all the downloady links.

### PR DESCRIPTION
**Pull request by hubgit**
Quoting @paulirish:

> Now that SSL is encouraged for everyone and doesn’t have performance concerns, this [protocol-relative] technique is now an anti-pattern. If the asset you need is available on SSL, then always use the https:// asset.

http://www.paulirish.com/2010/the-protocol-relative-url/

fixes #86

### CODE REVIEW

**Comment by arthurvr** [comment]
Hey @hubgit! Thanks for your pull request. It looks like you signed the CLA with a private GitHub email address, which we specifically don't allow: https://github.com/jquery/jquery-license/commit/fdb6b247448cd3369782e39708fff1d42a2f0e44. Could you please sign with your real email and update the git config? Thanks!

File: pages/download.md [hunk 1]
```diff
@@ -91,8 +91,8 @@ if the visitor to your webpage has already downloaded a copy of jQuery from the
 
 To use the jQuery CDN, just reference the file directly from `http://code.jquery.com` in the script tag:
```
Comment (arthurvr) [inline pages/download.md]: > just reference the file directly from `http://code.jquery.com` in the script tag:

Should we update this one as well?

Comment (arthurvr) [inline pages/download.md]: ping @hubgit. Would love to be able to land this.

**Comment by hubgit** [comment]
Updated to convert as many URLs as possible to HTTPS.

**Comment by hubgit** [comment]
@arthurvr This should be ok now?

**Comment by mzgol** [comment]
So, are we doing it? I think it's a good change. @kborchers, AFAIK you had some opinions here? :)

**Comment by kborchers** [comment]
As long as the links work I am fine with this change. The specific incident that got me involved was with a protocol-relative link to sizzlejs.com from https which caused the link to be broken as there is no cert for sizzlejs.com

**Comment by hubgit** [comment]
Updated to fix merge conflicts.

**Comment by dmethvin** [comment]
Fixed in e37ea3fa2fce05d0c9974150bbf45adad45d68ff .

**Pull request closed without merging**